src/sql/src/parsers/create_parser.rs

```diff
@@ -215,6 +209,14 @@ impl<'a> ParserContext<'a> {
                 }
             );
         }
+        if let Some(append_mode) = options.get("append_mode") {
+            if append_mode == "true" && options.contains_key("merge_mode") {
+                return InvalidDatabaseOptionSnafu {
+                    key: "merge_mode".to_string(),
+                }
+                .fail();
+            }
+        }
```

Comment:
I think we should not put this check on the parser side. Is it possible to move the check to the ddl phase?

Reply:
I don't think the parser should handle this business logic... But this isn't a blocker.